src/main/java/oracle/kubernetes/operator/helpers/ConfigMapHelper.java

```diff
@@ -1,143 +1,165 @@
-  }
-
-  @Override
-  public boolean containsValue(Object value) {
-    throw new UnsupportedOperationException();
-  }
-
-  @Override
-  public String get(Object key) {
-    if (key instanceof String) {
-      return readValue((String) key);
-    }
-    return null;
-  }
-
-  @Override
-  public String put(String key, String value) {
-    throw new UnsupportedOperationException();
-  }
-
-  @Override
-  public String remove(Object key) {
-    throw new UnsupportedOperationException();
-  }
-
-  @Override
-  public void putAll(Map<? extends String, ? extends String> m) {
-    throw new UnsupportedOperationException();
+  private ConfigMapHelper() {}
+  
+  /**
+   * Factory for {@link Step} that creates config map containing scripts
+   * @param namespace Namespace
+   * @param next Next processing step
+   * @return Step for creating config map containing scripts
+   */
+  public static Step createScriptConfigMapStep(String namespace, Step next) {
+    return new ScriptConfigMapStep(namespace, next);
   }
 
-  @Override
-  public void clear() {
-    throw new UnsupportedOperationException();
-  }
-
-  @Override
-  public Set<String> keySet() {
-    Set<String> keys = new HashSet<>();
-    String[] list = mountPointDir.list();
-    if (list != null) {
-      for (String s : list) {
-        keys.add(s);
-      }
+  private static class ScriptConfigMapStep extends Step {
+    private final String namespace;
+    
+    public ScriptConfigMapStep(String namespace, Step next) {
+      super(next);
+      this.namespace = namespace;
     }
-    return keys;
-  }
-
-  @Override
-  public Collection<String> values() {
-    throw new UnsupportedOperationException();
-  }
-
-  @Override
-  public Set<Entry<String, String>> entrySet() {
-    Set<Entry<String, String>> entries = new HashSet<>();
-    String[] list = mountPointDir.list();
-    if (list != null) {
-      for (String s : list) {
-        entries.add(new Entry<String, String>() {
 
-          @Override
-          public String getKey() {
-            return s;
+    @Override
+    public NextAction apply(Packet packet) {
+      String name = KubernetesConstants.DOMAIN_CONFIG_MAP_NAME;
+      V1ConfigMap cm = new V1ConfigMap();
+      
+      V1ObjectMeta metadata = new V1ObjectMeta();
+      metadata.setName(name);
+      metadata.setNamespace(namespace);
+      
+      AnnotationHelper.annotateWithFormat(metadata);
+      
+      Map<String, String> labels = new HashMap<>();
+      labels.put(LabelConstants.CREATEDBYOPERATOR_LABEL, "true");
+      metadata.setLabels(labels);
+
+      cm.setMetadata(metadata);
+
+      Map<String, String> data = new HashMap<>();
+      
+      data.put("livenessProbe.sh", 
+          "#!/bin/bash\n" + 
+          "# Kubernetes periodically calls this liveness probe script to determine whether\n" + 
+          "# the pod should be restarted. The script checks a WebLogic Server state file which\n" + 
+          "# is updated by the node manager.\n" + 
```

Comment:
It'd be nice to add a description of $1 and $2, or make it self documenting by adding a DOMAIN_UID=$1 and WL_SERVER_NAME=$2, and referencing the new names .  (domain-uid & wl-server-name respectively).

Ditto for readinessProbe.sh